modules/@apostrophecms/admin-bar/ui/apos/components/TheAposContextBar.vue

```diff
@@ -498,7 +498,7 @@ export default {
       const contextOptions = this.context
         ? apos.modules[this.context.type]
         : { contentChangedRefresh: true };
-      if (contextOptions.contentChangedRefresh) {
+      if (!e.isLocalizing && contextOptions.contentChangedRefresh) {
```

Comment:
When localizing a doc (different url than current page) we don't want the event listener to rewrite the url...
Don't really know why we do that in first instance.